assets/prefabs/potion/DeathPotion.prefab

```diff
@@ -0,0 +1,28 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Potions:DeathPotionBottle",
+        "stackId": "Potions:DeathPotion",
+        "consumedOnUse": true
+    },
+    "DisplayName": {
+        "name": "Death Potion",
+        "description": "Harms 100 HP."
+    },
+    "Potion": {
+        "hasGenome": false,
+        "effects": [
+            {
+                "effect": "Death",

```

Comment:
"Death" is not an existing effect, you still need to find the effect that harms the player in `PotionCommonEffects`.